app/src/main/java/org/fossasia/susi/ai/login/LoginActivity.kt

```diff
@@ -86,6 +95,7 @@ class LoginActivity : AppCompatActivity(), ILoginView {
             }
         }
         log_in.isEnabled = true
+        forgot_password.isEnabled = true
```

Comment:
Use camel case

Reply:
@iamareebjamal should I make another issue for the camel case replacement in the app, because this one has multiple uses in the same activity and they have no relation to this PR.

Reply:
OK, but I was referring to the new code. You should correct things one by one in at least new code. But yes, handle that in separate issue

Reply:
the issue I created is #1330